src/Component/Tests/Dummy/FooFightersGrid.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Component\Grid\Tests\Dummy;
+
+use Sylius\Bundle\GridBundle\Builder\GridBuilderInterface;
+use Sylius\Bundle\GridBundle\Grid\AbstractGrid;
+use Sylius\Bundle\GridBundle\Grid\ResourceAwareGridInterface;
+
+final class FooFightersGrid extends AbstractGrid implements ResourceAwareGridInterface
```

Comment:
Can we write one e2e test with usage of this grid?

Reply:
https://github.com/Sylius/SyliusGridBundle/pull/191/files#diff-19a4971649a951cf9ff2533feef055eca83186b5e7837011f6ff746717ca5ec9R694

Reply:
But this one is better
https://github.com/Sylius/SyliusGridBundle/pull/191/files#diff-19a4971649a951cf9ff2533feef055eca83186b5e7837011f6ff746717ca5ec9R692

Reply:
But this one is better https://github.com/Sylius/SyliusGridBundle/pull/191/files#diff-e5f9e1c0731852fd99ea54b11133276943457f440a8ff43f4ee9b4a4a1831915R23

Reply:
But we don't change anything there. Shouldn't we try to turn off some field and add another?

Reply:
@lchrusciel done https://github.com/Sylius/SyliusGridBundle/pull/191/commits/1e48c9a8b3d5fb57312bda7f66f0568fc55b7106